src/main/java/dev/openfeature/sdk/OpenFeatureClient.java

```diff
@@ -34,17 +34,13 @@ public class OpenFeatureClient implements Client {
     private EvaluationContext evaluationContext;
 
     /**
-     * Deprecated public constructor. Use OpenFeature.API.getClient() instead.
-     * 
+     * Create an OpenFeature client. For internal use only.
+     *
      * @param openFeatureAPI Backing global singleton
      * @param name           Name of the client (used by observability tools).
      * @param version        Version of the client (used by observability tools).
-     * @deprecated Do not use this constructor. It's for internal use only.
-     *             Clients created using it will not run event handlers.
-     *             Use the OpenFeatureAPI's getClient factory method instead.
      */
-    @Deprecated() // TODO: eventually we will make this non-public
-    public OpenFeatureClient(OpenFeatureAPI openFeatureAPI, String name, String version) {
+    OpenFeatureClient(OpenFeatureAPI openFeatureAPI, String name, String version) {
```

Comment:
This has bee deprecated for a while, but technically is a breaking change. I think we need to wait for a 2.0 for this.

@thomaspoignant @lukas-reining @kinyoklion is there any reason we should make this an exception?

Reply:
My expectation would be to wait until 2.0. We should probably be tracking this with an issue and "breaking-change" or other label (or project/mileston). So when we want to do 2.0 we can go through that list.

Reply:
Thanks @toddbaert  @kinyoklion  for feedback. I will document this and revert the change

Reply:
Created tracking issue #872 and reverted this change

Reply:
May want to update PR title. Aside from that LGTM.

Reply:
@kinyoklion ahh good one, thanks 👍